src/applications/disability-benefits/new-686/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "New 686",
+  "entryFile": "./app-entry.jsx",
+  "entryName": "new-686",
+  "rootUrl": "/disability-benefits/new-686",
+  "production": false
```

Comment:
I'm not sure if the top level `production` property here is considered deprecated or not. Changing this to `template.vagovprod: false` might be safer.

Reply:
fixed